modules/gateway/persist.go

```diff
@@ -19,30 +19,81 @@ const (
 // gateway persist file.
 var persistMetadata = persist.Metadata{
 	Header:  "Sia Node List",
+	Version: "0.4.0",
```

Comment:
Metadata version should only be bumped if compatibility is broken with the previous file. And, I'm pretty sure that we can pull this off without breaking compatibility. Further, when we do increment the version, we increment it to the version of Sia that it's going to be released in, so we would set the new version to `"1.2.1"` in this case.

But, as long as the file is already JSON (which it is), we should be able to preserve compatibility by adding new named fields to correspond with the new features. They will be read in as blank, and so set to default values.